docs/source/markdown/podman-machine-reset.1.md

```diff
@@ -0,0 +1,49 @@
+% podman-machine-reset 1
+
+## NAME
+podman\-machine\-reset - Reset Podman machines and environment
+
+## SYNOPSIS
+**podman machine reset** [*options*]
+
+## DESCRIPTION
+
+Reset your podman machine environment.  This command stops any running machines
+and then removes them.  Configuration and data files are then removed.  Data files
+would include machine disk images and any previously pulled cache images.  When
+this command is run, all of your Podman machines will have been deleted.
```

Comment:
```suggestion
this command is run, all of the Podman machines on the host will have been deleted.
```

or perhaps "in the environment" rather than "on the host".